adapters/lora-vertx/src/main/java/org/eclipse/hono/adapter/lora/LoraCommandProperties.java

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+ * Copyright (c) 2016, 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.adapter.lora;
+
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Properties for LoRaWAN commands.
+ */
+public class LoraCommandProperties {
+
+    private final List<String> commandEnabledTenants = new LinkedList<>();
+
+    public List<String> getCommandEnabledTenants() {
```

Comment:
A javadoc comment would be good here.

The fact that the Command&Control feature of the LoRa adapter is restricted to these tenants should be pointed out in the (to be written) documentation and could also be mentioned in the commit comment.